cpp/ql/lib/semmle/code/cpp/ir/dataflow/internal/DataFlowUtil.qll

```diff
@@ -2232,6 +2232,9 @@ private module Cached {
       // Def-use/Use-use flow
       Ssa::ssaFlow(nodeFrom, nodeTo)
       or
+      // Phi input -> Phi
+      nodeFrom.(SsaPhiInputNode).getPhiNode() = nodeTo.(SsaPhiNode).getPhiNode()
+      or
```

Comment:
This looks good, but I'm now starting to wonder how flow into Phi nodes was modelled before? Was that just the operand -> instruction flow?

Reply:
Flow into phis was previously handled in `useToNode` here: https://github.com/github/codeql/blob/ac6cc38b2097e5a26160962d1212a13bef702c04/cpp/ql/lib/semmle/code/cpp/ir/dataflow/internal/SsaInternals.qll#L667-L671